docs/src/main/tut/reference.md

```diff
@@ -38,26 +38,26 @@ _SQL_: `rf_reproject_geometry`
 Reproject the vector `geom` from `origin_crs` to `destination_crs`. Both `_crs` arguments are either [proj4](https://proj4.org/usage/quickstart.html) strings, [EPSG codes](https://www.epsg-registry.org/) codes or [OGC WKT](https://www.opengeospatial.org/standards/wkt-crs) for coordinate reference systems. 
 
 
-#### envelope
+#### rf_extent
```

Comment:
Now that many of our python functions have the same name as the SQL function registered, we may be able to pull out the SQL v python bits in these. Decent amount of stubby pencil work but i'm happy to help with it.

Reply:
I'd rather spend the time to generate this file from annotations ;-) I'm mulling over an approach.